e2etests/tests/graceful_restart.go

```diff
@@ -128,8 +142,33 @@ var _ = ginkgo.Describe("Establish BGP session with EnableGracefulRestart", func
 			Consistently(check, 2*time.Minute, time.Second).ShouldNot(HaveOccurred())
 			Eventually(c, time.Minute, time.Second).Should(BeClosed(), "restart FRRK8s pods are not yet ready")
 		},
-			ginkgo.Entry("IPV4", ipfamily.IPv4, "192.168.2.0/24"),
-			ginkgo.Entry("IPV6", ipfamily.IPv6, "fc00:f853:ccd:e799::/64"),
+			ginkgo.Entry("IPV4", ipfamily.IPv4, "192.168.2.0/24", "200.200.200.0/24"),
+			ginkgo.Entry("IPV6", ipfamily.IPv6, "fc00:f853:ccd:e799::/64", "2001:db8::/64"),
 		)
 	})
 })
+
+// checkPrefixOnNodes checks that a prefix has at least one route on the
```

Comment:
should this belong to some pkg?

Reply:
Imo this help function is tailored  to the graceful restart scenario because it checks  the routes on the host itself (all other test use the routes from the vtysh) and is relaxed on the next hop (because not all nexthops exist as decribed int he coment). So I would not transfer into a pkg.

No objection though if you want that (which pkg do you have in mind)?